### ISSUE TITLE
Usage of cgalConcept

### ISSUE THREAD

**Issue by albert-github** [CGAL/cgal#3551]
## Issue Details

In the documentation in (e.g.) the concept description the `\brief` construct is used with some long texts and commands that break the (XHTML) layout. The `\brief` is intended for small compact messages.

We can see in the master version (at the moment of writing this issue) for the Solver  interface in the file:
`annotated.html`:
![master_annotated](https://user-images.githubusercontent.com/5223533/50089315-cb468800-0205-11e9-8be7-d9245f159246.jpg)

`group__PkgSolverInterfaceConcepts.html`:
![master_group](https://user-images.githubusercontent.com/5223533/50089365-f4671880-0205-11e9-814d-b510b49c24d7.jpg)

`classDiagonalizeTraits.html`:
![both_traits](https://user-images.githubusercontent.com/5223533/50089346-e74a2980-0205-11e9-901a-679f21f750b2.jpg)

The first 2 are extensive brief descriptions.

Due to a change in doxygen (as of December 16, 2018) we get the following:
![new_annotated](https://user-images.githubusercontent.com/5223533/50089502-4ad45700-0206-11e9-9e13-de17007082ce.jpg)

![new_group](https://user-images.githubusercontent.com/5223533/50089513-4f990b00-0206-11e9-83b4-d4b4a328485e.jpg)

![both_traits](https://user-images.githubusercontent.com/5223533/50089519-51fb6500-0206-11e9-93bd-cea7569b402e.jpg)

In the first 2 the text:
`For example, a matrix of dimension 3 is defined as follows: `
is a bit strange now.

When looking in the file:
`Solver_interface/doc/Solver_interface/Concepts/DiagonalizeTraits.h`
we see (line 6 ff):
```
from covariance matrices represented by an array `a`, using symmetric
diagonalization. For example, a matrix of dimension 3 is defined as
follows:
<center>
```

**Suggestion**
It is suggested to change this in:
```
from covariance matrices represented by an array `a`, using symmetric
diagonalization.

For example, a matrix of dimension 3 is defined as
follows:
<center>

```

The results are:
![suggested_annotated](https://user-images.githubusercontent.com/5223533/50089770-0bf2d100-0207-11e9-9a5d-ba42bb065926.jpg)

![suggested_group](https://user-images.githubusercontent.com/5223533/50089774-0dbc9480-0207-11e9-9843-8e730798962d.jpg)

![suggested_traits](https://user-images.githubusercontent.com/5223533/50089779-0eedc180-0207-11e9-9a1e-667ada8f0f66.jpg)

**Comment by albert-github** [comment]
Well I have some more knowledge about the internals of doxygen and other direct communication channels.

I was seeing / investigating if there were some problems with doxygen in respect to XHTML and used CGAL as test. During the test I found that the cgalConcept resulted in some non XHTML conform code. To solve this e.g. the `<center>` is not allowed in a brief description anymore (committed on the 16-th).
After the fix I rebuild the CGAL documentation to see how it looked and so I saw the small problem.
In HTML tests the problem does not surface as no warning or error is thrown.
